Ease.elm

```diff
@@ -7,34 +22,39 @@ All easing functions expect inputs to be bewteen zero and one, and will typicall
 happens at the start of the transition, easing "out" at the end, and "inOut" on both sides. The functions provided here
 are meant to match the graphical examples on [easings.net](http://easings.net/).
 
-```elm
-import Ease
-n = 10
+    import Ease
+    n = 10
 
-List.map (\i -> Ease.inQuad (i/n)) [0..n]
-> [0, 0.01, 0.04, 0.09, 0.16, 0.25, 0.36, 0.49, 0.64, 0.81, 1]
+    List.map (\i -> Ease.inQuad (i/n)) [0..n]
```

Comment:
Replace with `List.range`.

Consider using elm-verify-examples which catches this sort of thing.

Reply:
I fixed the example in the comment. `elm-verify-examples` is not yet published for 0.19.